apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -388,6 +388,40 @@ check_deps_and_remove(ConfRootKey, BridgeType, BridgeName, AlsoDeleteActions) ->
 %% Helpers for CRUD API
 %%--------------------------------------------------------------------
 
+maybe_withdraw_rule_action(ConfRootKey, BridgeType, BridgeName, DeleteDeps) ->
```

Comment:
Nit. Does "withdraw" simply means "remove" in this context? If so, maybe just use "remove" then?

Reply:
I assume so.  I just "forked" this from the previous `emqx_bridge_lib:maybe_...`, we could rename both later.